recipes/parquet-python/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "parquet-python" %}
+{% set version = "1.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/p/parquet/parquet-{{ version }}.tar.gz
+  sha256: fb1c90768c1b9159d4d6a9b3112ea8107b0b46d7491c5ac452ba7350f333bf0a
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  entry_points:
+    - parquet = parquet.__main__:main',
```

```suggestion
    - parquet = parquet.__main__:main
```
Function names can't contain punctuation.